File: youtubejextractor/src/main/java/com/github/kotvertolet/youtubejextractor/models/test/ServiceTrackingParamsItem.java
package com.github.kotvertolet.youtubejextractor.models.test;

import java.util.List;
import com.google.gson.annotations.SerializedName;

public class ServiceTrackingParamsItem{

	@SerializedName("service")
	private String service;

	@SerializedName("params")
	private List<ParamsItem> params;

	public void setService(String service){
		this.service = service;
	}

	public String getService(){
		return service;
	}

	public void setParams(List<ParamsItem> params){
		this.params = params;
	}

	public List<ParamsItem> getParams(){
		return params;
	}
}

File: youtubejextractor/src/main/java/com/github/kotvertolet/youtubejextractor/models/newModels/ServiceTrackingParamsItem.kt
package com.github.kotvertolet.youtubejextractor.models.newModels

import com.google.gson.annotations.SerializedName

class ServiceTrackingParamsItem {
    @SerializedName("service")
    var service: String? = null

    @SerializedName("params")
    var params: List<ParamsItem>? = null
    override fun toString(): String {
        return "ServiceTrackingParamsItem{" +
                "service = '" + service + '\'' +
                ",params = '" + params + '\'' +
                "}"
    }
}